webapp/app/partials/perf/perfctrl.html

```diff
@@ -16,6 +16,7 @@
             <span class="sr-only">Remove series</span>
           </button>
           <table>
+            <input title="Show/Hide series" type="checkbox" id="showHideCheck" ng-click="hideSeries(series.signature)" checked>
```

Comment:
Trivial, but I think we've been trying to capitalize just the first letter for Title consistency, unless it's using a specific platform term. The other Show/hide I could think of is here as an example. I'm easy though, maybe we update that one instead.
https://github.com/mozilla/treeherder-ui/blob/2f7f9d92f5bcbdaf62d232447f787adbbbfa169e/webapp/app/partials/main/thWatchedRepoNavPanel.html#L18